_maps/map_files/Galactica/job_changes.dm

```diff
@@ -0,0 +1,7 @@
+#define JOB_MODIFICATION_MAP_NAME "NSV Galactica"
+
+/datum/job/munitions_technician/New()
+	..()
+	MAP_JOB_CHECK
+	total_positions = 10
+	spawn_positions = 10
```

Comment:
Needs to be included through special means.

See [code/modules/jobs/map_changes/map_changes.dm](../../blob/master/code/modules/jobs/map_changes/map_changes.dm)

Reply:
Though I might test recoding that entire system tomorrow